modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -1196,9 +1196,21 @@ public CodegenOperation fromOperation(String path, String httpMethod, Operation
                 // empty list, do nothing to override global setting
             }
         } else if (swagger != null && swagger.getConsumes() != null && swagger.getConsumes().size() > 0) {
-            // use consumes defined globally 
-            consumes = swagger.getConsumes();
-            LOGGER.debug("No consumes defined in operation. Using global consumes (" + swagger.getConsumes() + ") for " + op.operationId);
+            Boolean hasFormData = false;
+            for (Parameter param : operation.getParameters()) {
+                if (param.getIn().equals("formData")) {
+                    hasFormData = true;
+                }
+            }
+            if (hasFormData) {
+                consumes.add("multipart/form-data");
```

Comment:
Technically it can also be `application/x-www-form-urlencoded`.

Instead of adding a default, should the spec owner fix it by adding a proper MIME instead?